vendir.yaml

```diff
@@ -47,3 +47,32 @@ test:
       runs: |
         cd examples/inline
         vendir sync
+    - name: test help command
+      runs: "# Test command help \nvendir --version\nvendir -h\nvendir sync -h\n"
```

Comment:
Having this in a single line seems rather strange and is less readable to me.

Reply:
This is yam.  I prompted it to fix it, and it was reverted by the linter 🙃